vignettes/Demography.Rmd

```diff
@@ -15,24 +15,38 @@ knitr::opts_chunk$set(
 ```
 
 ```{r setup}
-suppressPackageStartupMessages(library(ggplot2))
+# Load in the requisite package(s):
 library(malariasimulation)
 ```
 
-# Parameterisation
+# Introduction
 
-We are going to create a scenario where we track severe prevalence for 5 years.
+The dynamics of malaria transmission, and the efficacy of interventions designed to interrupt it, are highly context specific and an important consideration is the demography of the population(s) under investigation. To suit the research needs of the user, `malariasimulation` allows for the specification of custom human population demographics by allowing them to specify age-group specific death rates parameters. It also enables users to instruct the model to render outputs with variables of interest (e.g. human population, number of cases detectable by microscopy, number of severe cases, etc.) presented by user-defined age groups. In this vignette we'll use simple, illustrative cases to demonstrate how to customise the output rendered by the `run_simulation()` function, specify both fixed and time-varying custom death rates.
```

Comment:
last sentence "and how to specify..."

Reply:
Corrected the the last sentence to: 

"In this vignette we'll use simple, illustrative cases to demonstrate how to customise the output rendered by the `run_simulation()` function, and how to specify both fixed and time-varying custom death rates."